assets/js/aplus.js

```diff
@@ -86,6 +86,37 @@ $(function() {
 
     $(window).bind('scroll', modifyMenu);
     $(window).bind('resize', updateMenu);
+
+    /**
+    * Warn about links that open in new windows.
+    */
+
+    function addLinkType(link, type) {
+      let linkTypes = (link.getAttribute('rel') || '').split(' ');
+      if (!linkTypes.includes(type)) {
+        linkTypes.push(type);
+      }
+      link.setAttribute('rel', linkTypes.join(' ').trim());
+    }
+
+    function addExternalLinkIcon(link) {
+      if (!link.querySelector('.icon')) {
+        link.insertAdjacentHTML('beforeend', `<span class="icon external-link"></span>`);
+      }
+    }
+
+    function addScreenReaderMessage(link, message) {
+      if (!link.querySelector('.screen-reader-only')) {
```

Comment:
Why did you make a new class `screen-reader-only` instead of using the `sr-only` in Bootstrap?

Reply:
I forgot about that class. I will use `sr-only` instead